owca/testing.py

```diff
@@ -103,15 +103,16 @@ def anomaly(contended_task_id: TaskId, contending_task_ids: List[TaskId],
     )
 
 
-def task(cgroup_path, labels=None, resources=None):
+def task(cgroup_path, labels=None, resources=None, subcgroups_paths=None):
     """Helper method to create task with default values."""
     prefix = cgroup_path.replace('/', '')
     return Task(
         cgroup_path=cgroup_path,
         name=prefix + '_tasks_name',
         task_id=prefix + '_task_id',
         labels=labels or dict(),
-        resources=resources or dict()
+        resources=resources or dict(),
+        subcgroups_paths=subcgroups_paths if subcgroups_paths is not None else []
```

Comment:
Can't it be:
```suggestion
        subcgroups_paths=subcgroups_paths or []
```
as it is in the line above?

Reply:
Yes, we should make it like that.